express/parsers/formats/zip.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import
+
+import os
+import zipfile
+
+
+class BaseZipParser(object):
+    """
+    Base Zip parser class.
+    """
+
+    def __init__(self, zip_file_path):
+        self.zip_path = zip_file_path
+        self.zip_dir_name = None
+        self.zip_file = None
+        if self.zip_path and os.path.exists(self.zip_path):
+            try:
+                self.zip_dir_name = os.path.dirname(self.zip_path)
+                self.zip_file = zipfile.ZipFile(self.zip_path)
+            except:
+                # safely ignore broken zip file
```

Comment:
Let's log the event here, pls, to avoid a silent loophole

Reply:
I did not like this either, but followed this pattern:
https://github.com/Exabyte-io/express/blob/a8a9cd27433a52c6c764f4f61286910fc1924d55/express/parsers/formats/xml.py#L20-L22